.circleci/config.yml

```diff
@@ -40,7 +40,14 @@ workflows:
       # -- PR check jobs ------------------------------------------
       - kurtosis-docs-checker/check-docs:
           should-check-changelog: false
-          markdown-link-check-config-json: "{}"
+          markdown-link-check-config-json: |
+            {
+                "ignorePatterns": [
+                    {
+                        "pattern": "https://github.com/kurtosis-tech/near-package"
+                    }
+                ]
+            }
```

Comment:
Why are we doing that?

Reply:
the release please thing links to itself, to compare between the last released versions and the docs-checker complains about it